workflows/scripts/03_export_spike2.py

```diff
@@ -6,21 +6,50 @@
 from trialexp.process.pycontrol.utils import export_session
 from snakehelper.SnakeIOHelper import getSnake
 from workflows.scripts import settings
+from glob import glob 
+from trialexp.process.pyphotometry.utils import *
 
 #%%
 
 (sinput, soutput) = getSnake(locals(), 'workflows/spout_bar_nov22.smk',
     [settings.debug_folder +'\processed\spike2.smrx'],
   'export_spike2')
 
+#%% Photometry dict
+
+fn = glob(sinput.photometry_folder+'\*.ppd')[0]
```

Comment:
`fn` may be empty if no photometry data is found. We will need to handle the error here and only export the pycontrol data when no photometry data is found

Reply:
> Also, suggest to use a Path object to make sure the code can work both in Windows and Linux, e.g. `fn = list(Path(sinput.photometry_folder).glob('*.ppd'))[0]`

Uh-huh, that's the secret of why the random use of file separators is accepted!